src/validation.cpp

```diff
@@ -1799,8 +1799,8 @@ bool CChainState::ConnectBlock(const CBlock& block, BlockValidationState& state,
         for (const auto& tx : block.vtx) {
             for (size_t o = 0; o < tx->vout.size(); o++) {
                 if (view.HaveCoin(COutPoint(tx->GetHash(), o))) {
-                    return state.Invalid(BlockValidationResult::CONSENSUS, error("ConnectBlock(): tried to overwrite transaction"),
-                                     "bad-txns-BIP30");
+                    LogPrintf("ConnectBlock(): tried to overwrite transaction\n");
```

Comment:
In commit "[validation] Remove error() calls from Invalid() calls" (1550f781fc0a86fd318883512a7104d018cf7c56)

Is it intentional to drop the `ERROR:` prefix here and elsewhere in this PR?

https://github.com/bitcoin/bitcoin/blob/5d2ccf0ce9ca1571c650a69319fb9c1e0b626ecb/src/util/system.h#L60-L65

If so, it would be good to mention the change in behavior in the commit message, and maybe in release notes.

Reply:
I can't remember whether it was intentional or not. I've re-added the `ERROR: ` prefix so there's no change in logging behaviour.